src/org/opensolaris/opengrok/web/Util.java

```diff
@@ -1234,17 +1234,17 @@ public static void printMessages(Writer out, SortedSet<Message> set, boolean lim
      * @return json array containing the set of messages
      */
     @SuppressWarnings("unchecked")
-    public static JSONArray messagesToJson(SortedSet<Message> set) {
+    public static JSONArray messagesToJson(SortedSet<AcceptedMessage> set) {
         JSONArray array = new JSONArray();
         SimpleDateFormat df = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss z");
-        for (Message m : set) {
+        for (AcceptedMessage m : set) {
```

Comment:
aren't we using gson now?
call your encode/decode on the message?

Reply:
but keep in mind this will break the ui as it needs to be adjusted too

Reply:
Okay, I will try to do it :)

Reply:
you can start with using gson (to replace the json simple) and the ui changes can come in another pr